charts/civil-ccd/values.preview.template.yaml

```diff
@@ -48,10 +49,78 @@ ccd:
       environment:
         OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
         CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: civil_service,aac_manage_case_assignment
+        ELASTIC_SEARCH_ENABLED: true
+        ELASTIC_SEARCH_HOSTS: ${SERVICE_NAME}-es-master:9200
+        ELASTIC_SEARCH_DATA_NODES_HOSTS: http://${SERVICE_NAME}-es-master:9200
       ingressHost: ccd-data-store-api-${SERVICE_NAME}.preview.platform.hmcts.net
   ccd-definition-store-api:
     java:
       ingressHost: ccd-definition-store-${SERVICE_NAME}.preview.platform.hmcts.net
+      environment:
+        DEFINITION_STORE_DB_HOST: ${SERVICE_NAME}-postgresql
+        DEFINITION_STORE_IDAM_KEY: ${CCD_DEFINITION_STORE_S2S_SECRET}
+        IDAM_USER_URL: https://idam-web-public.aat.platform.hmcts.net
+        ELASTIC_SEARCH_ENABLED: true
+        ELASTIC_SEARCH_HOST: ${SERVICE_NAME}-es-master
+        ELASTIC_SEARCH_CASE_INDEX_MAPPING_FIELDS_LIMIT: 10000
 camunda-bpm:
   java:
     ingressHost: camunda-${SERVICE_NAME}.preview.platform.hmcts.net
+
+elastic:
+  enabled: true
+
+elasticsearch:
+  nameOverride: ${SERVICE_NAME}-es
+  clusterName: "es"
+  imageTag: "7.17.1"
+  replicas: 1
+  minimumMasterNodes: 1
+  antiAffinity: "soft"
+  esJavaOpts: "-Xmx512m -Xms512m"
+  initResources:
+    limits:
+      cpu: "500m"
+      memory: "512Mi"
+    requests:
+      cpu: "25m"
+      memory: "512Mi"
+  persistence:
+    enabled: false
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: traefik
+    path: /
+    hosts:
+      - es-${SERVICE_FQDN}
+
+logstash:
+  image:
+    repository: hmctspublic.azurecr.io/ccd/logstash
+    tag: latest
+    pullPolicy: Always
+  logstashJavaOpts: -Xmx1g -Xms512m
+  resources:
+    limits:
+      cpu: 2500m
+      memory: 1024Mi
+    requests:
+      cpu: 500m
+      memory: 1024Mi
+  persistence:
+    enabled: false
+  volumeMounts:
+    - name: pipeline
+      mountPath: /usr/share/logstash/files
+  elasticsearch:
+    host: ${SERVICE_NAME}-es-master
+    port: 9200
+  configTpl:
+    xpack.monitoring.enabled: "false"
+    db.url: jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified
+    db.user: ${POSTGRES_USER}
+    db.pwd: ${POSTGRES_PASSWORD}
+    es.data.nodes.url: http://${SERVICE_NAME}-es-master:9200
+  livenessProbe:
+    initialDelaySeconds: 90
```

Comment:
This is probably no longer needed thanks to Max's changes.

Try using the `pr-values:elasticsearch` label in the github PR and see if that works for you (of course after updating the code).